monkey/monkey_island/cc/models/test_monkey.py

```diff
@@ -6,8 +6,8 @@
 
 from monkey_island.cc.models.monkey import Monkey, MonkeyNotFoundError
 
-from .monkey_ttl import MonkeyTtl
 from ..test_common.fixtures import FixtureEnum
+from .monkey_ttl import MonkeyTtl
```

Comment:
Do we want relative imports?

Reply:
No, but I think it's out of scope to fix them now.